cloudformation/template.yaml

```diff
@@ -39,11 +68,6 @@ Parameters:
     Description: The name for your Cognito Identity Pool.
     Default: "DevPortalUserPool"
 
-  AWSRegion:
```

Comment:
Yep! Unnecessary/unused, and highly confusing for SAR users (since it's shown in the console in SAR). Thanks for the removal. :)